### ISSUE TITLE
[Bug]: OceanBlock Teams not working

### ISSUE THREAD

**Issue by whitesphinx** [FTBTeam/FTB-Modpack-Issues#149]
### Modpack

FTB Oceanblock

### Modpack version

1.3.0

### Describe the bug

I am playing a solo map, opening up Lan so that my friend and I can connect and play the Oceanblock mod pack together. we noticed that the Teams feature in that mod pack is not working, I am not sure if this is just not ready yet or if it is a bug, but it really makes it difficult because when we complete achievements we have to toss items back and fourth also I had to enable the claimed chunks to allow for him to interact with anything. 

### Steps to reproduce

log into game,
load world,
open to Lan
wait for friend to connect.
attempt to add him to the team so that he can play, 
he doesn't even show up in the game.

### Expected behaviour

I expect to be able to add/invite my friend to my team so that we can complete missions and interact with the island with out having to do a bunch of extra steps.

**Comment by whitesphinx** [comment]
So we did manage to get on the same team, but it did not work as smoothly
as u suggest. For some reason with the world being hosted by me. He was
unable to leave his team we tried many times different ways for him to
leave his team and it just wasn't working, also when he tried to send me an
invite to his team, since he couldn't leave his team, it would never show
up in the chat or anything I never saw the invite. I just typed in the
command to join this team and that worked I guess. Also any time I tried to
invite it never showed up in the chat for him to accept the invite.
Something is bugged, not sure what it is but we did manage to get on the
same team, at the loss of quest progress and the loss of both of our entire
inventories.apparently when u leave a team ur entire inventory vanishes.
